Tools/TestWebKitAPI/Tests/WebKitCocoa/WKWebsiteDatastore.mm

```diff
@@ -448,4 +448,10 @@ HTTPServer server([connectionCount = 0] (Connection connection) mutable {
     EXPECT_FALSE([WKWebsiteDataStore _defaultDataStoreExists]);
 }
 
+TEST(WebKit, DefaultHSTSDirectory)
+{
+    auto configuration = adoptNS([[_WKWebsiteDataStoreConfiguration alloc] init]);
+    EXPECT_GT(configuration.get().hstsStorageDirectory.absoluteString.length, 0U);
```

Comment:
Can we assert the url non-null (e.g. using `EXPECT_FALSE`)?

Reply:
Done!